GeneratorInterface/Pythia8Interface/plugins/ReweightUserHooks.h

```diff
@@ -26,6 +26,34 @@ class PtHatReweightUserHook : public Pythia8::UserHooks
     double pt, power;
 };
 
+class PtHatEmpReweightUserHook : public Pythia8::UserHooks
+{
+  public:
+   PtHatEmpReweightUserHook(double _pTHatMin, double _pTHatMax) :
+      pTHatMin(_pTHatMin), pTHatMax(_pTHatMax) {
+      sigma = TF1("sigma", "([0]*pow(x,[2]+[3]*log(0.01*x)+[4]*pow(log(0.01*x),2))*pow(1-2*x/(13000.+[5]),[1]))*x", pTHatMin, pTHatMax);
```

Comment:
hi @aperloff - please just make this a C++ function. Everything is hardwired anyway, so no need for the overhead of ROOT's TF1.